app/web/components/ImageInput/ImageInput.test.tsx

```diff
@@ -347,10 +354,13 @@ describe("ImageInput http error tests", () => {
   });
 
   it("displays a general error for server errors", async () => {
-    server.use(
-      rest.post("https://example.com/upload", async (_req, res, ctx) => {
-        return res(ctx.status(500), ctx.text("Internal server error"));
-      })
+    jest.spyOn(global, "fetch").mockImplementation(
+      async () =>
+        new Response(JSON.stringify({ error: "Internal server error" }), {
+          status: 500,
+          statusText: "Internal server error",
+          headers: { "Content-Type": "application/json" },
+        })
     );
```

Comment:
I dunno why the old way stopped working with the upgrade, but since this was using fetch, mocking fetch itself seemed a better way to write the test anyway as it's closer to how it works IRL.